elastic/kibana
# [Discover] Fix csv report for filtered discover result on dashboard

## Summary

Fixes #119452

This PR takes into consideration discover filters when generating csv report.

### Test notes
- Open discover
- Create saved search with columns, apply filter
- Create dashboard with saved search
- Generate csv report via panel actions
- Result should be filtered
